### PR TITLE
docs: update template-from-scratch

### LINKED ISSUE #104: docs: update template-from-scratch
update <https://coder.com/docs/tutorials/template-from-scratch> with
 
- confirmed steps
- new screenshots
- consistent md

### PR DESCRIPTION
**Pull request by EdwardAngert**
closes https://github.com/coder/internal/issues/104

- [x] update screenshots
- [x] consistent md
- [x] confirm and update `Dockerfile` and `main.tf`
   - matched up to the Docker Containers starter template
- upload template
   - [x] confirm CLI steps
   - [x] add steps in tab for Dashboard
- [x] ~confirm that the expand works~ it doesn't

<https://coder.com/docs/@template-from-scratch-update/tutorials/template-from-scratch>

### CODE REVIEW

**Review by bpmct — APPROVED** [review]

**Review by matifali — APPROVED** [review]

**Comment by EdwardAngert** [comment]
waiting for confirmation of [upload through UI](https://coder.com/docs/@template-from-scratch-update/tutorials/template-from-scratch#add-the-template-files-to-coder) steps before merge

**Comment by EdwardAngert** [comment]
note that this PR now includes a change to the UI https://github.com/coder/coder/pull/15101/commits/0141696be0c7651da4a590d65a58199b9124684e

**Pull request merged**
